wpilib/wpilib/drive/robotdrivebase.py

```diff
@@ -52,24 +52,23 @@ def __init__(self):
         self.setSafetyEnabled(True)
         self.setName("RobotDriveBase")
 
-    def setDeadband(self, deadband):
-        """Change the value for deadband scaling. The default value is `.02`. Values smaller than the
-         deadband are set to 0, while values larger than the deadband are scaled from 0.0 to 1.0. See
-         :meth:`applyDeadband()`
-
-         :param deadband: The deadband to set
-         :type deadband: float
-         """
+    def setDeadband(self, deadband: float):
+        """Change the value for deadband scaling.
+        
+        The default value is :data:`kDefaultDeadband`. Values smaller than the deadband are set to 0,
+        while values larger than the deadband are scaled from 0.0 to 1.0. See :meth:`applyDeadband`.
+
+        :param deadband: The deadband to set
+        """
         self.deadband = deadband
 
-    def setMaxOutput(self, maxOutput):
+    def setMaxOutput(self, maxOutput: float):
         """Configure the scaling factor for using drive methods with motor controllers in a mode
         other than PercentVbus or to limit the maximum output.
 
-        The default value is `1.0`
+        The default value is :data:`kDefaultMaxOutput`.
```

Comment:
In my (limited) experience, this doesn't resolve and just shows "kDefaultMaxOutput". Am I wrong?

Reply:
You are indeed wrong. It links fine.

Reply:
Does it just link or will it replace it with '1.0'

Reply:
It doesn't show the value. Although I just realised the correct directive would be `:const:`, but not even that shows the value by default. We can easily show the value instead, but we'd have to remember to change it if it changes.